File: 프로그래머스/lv0/120837. 개미 군단/개미 군단.java
class Solution {
    public int solution(int hp) {
        // 5 3 1
        int ant1 = 5;
        int ant2 = 3;
        int ant3 = 1;
        int sum = 0;
        
        while (hp >= ant1) {
            hp -= ant1;
            sum++;
        }
        while (hp >= ant2) {
            hp -= ant2;
            sum++;
        }
        while (hp >= 1) {
            hp -= ant3;
            sum++;
        }
        return sum;
    }
}

File: 프로그래머스/0/120837. 개미 군단/개미 군단.py
def solution(hp):
    # 5 3 1
    answer = 0
    while hp != 0:
        if hp >= 5:
            hp -= 5
            answer += 1
            continue
        if hp >= 3:
            hp -= 3
            answer += 1
            continue
        if hp >= 1:
            hp -= 1
            answer += 1
    return answer